examples/docker/solara/app.py

```diff
@@ -1,31 +1,111 @@
+import requests
```

Comment:
let's add a more descriptive name: 

![image](https://github.com/ploomber/doc/assets/989250/97921005-9557-4d94-bea1-7cb737c70331)


and also include a brief explanation of what the app does in the app itself (you can take that from the readme and jus summarize it)

Reply:
[Fixed](https://autumn-voice-6639.ploomberapp.io/)